### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -551,7 +551,7 @@ Soft deleted users: These users will be displayed in the output of Get-CsOnlineU
 
 **Updates in Teams PowerShell Module version 3.0.0 and above**
 
-The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOD environments starting March 31, 2023.):
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:554]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOD environments starting March 31, 2023.):
```

**Review by JohanFreelancer9 — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ Accept wildcard characters: False
 ### -Identity
 Indicates the Identity of the user account to be retrieved.
 
-For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
+For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOC environments starting March 31, 2023.):
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:145]:
```suggestion
For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments including GCC at present, and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -272,7 +272,7 @@ Get-CsOnlineUser -Filter {ExternalAccessPolicy -ge "xyz_policy"}
 
 The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later (excluding updates mentioned previously for Teams PowerShell Module version 5.0.0):
 
-In the Teams PowerShell Module version 3.0.0 or later, filtering functionality is now limited to the following attributes (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
+In the Teams PowerShell Module version 3.0.0 or later, filtering functionality is now limited to the following attributes (note that these changes are only rolled out in commercial environments at present and will be applicable to the latest TPM versions in GCCH and DOC environments starting March 31, 2023.):
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:275]:
```suggestion
In the Teams PowerShell Module version 3.0.0 or later, filtering functionality is now limited to the following attributes (note that these changes are only rolled out in commercial environments including GCC at present, and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md (hunk 1 above)
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:554]:
```suggestion
The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 3.0.0 and later, excluding updates mentioned previously for Teams PowerShell Module version 5.0.0 (note that these changes are only rolled out in commercial environments including GCC at present and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):
```

**Review by dariomws — CHANGES_REQUESTED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -269,10 +269,11 @@ Get-CsOnlineUser -Filter {UserPrincipalName -gt/-le/-lt “abc”}
 ```
 Get-CsOnlineUser -Filter {ExternalAccessPolicy -ge "xyz_policy"}
 ```
+**Updates in Teams PowerShell Module version 3.0.0 and abve**
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:272]:
```suggestion
**Updates in Teams PowerShell Module version 3.0.0 and above**
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -142,7 +142,7 @@ Accept wildcard characters: False
 ### -Identity
 Indicates the Identity of the user account to be retrieved.
 
-For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments at present and will be applicable to the lastest TPM versions in government environments starting March 31, 2023.):
+For TeamsOnly customers using the Teams PowerShell Module version 3.0.0 or later, you use the following values to identify the account (note that these changes are only rolled out in commercial environments including GCC at present, and will be applicable to the latest TPM versions in GCC High and DoD environments starting March 31, 2023.):
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:145]: March 31 or 15?

Comment (isrumnon) [reply skype/skype-ps/skype/Get-CsOnlineUser.md:145]: it's March 15 now

Comment (dariomws) [reply skype/skype-ps/skype/Get-CsOnlineUser.md:145]: We need to update it also in line 145

**Review by dariomws — APPROVED** [review]

**Comment by isrumnon** [comment]
@JohanFreelancer9  I updated a few other things, not sure why build is failing

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 I updated a few other things, not sure why build is failing

@yogkumgit

**Comment by yogkumgit** [comment]
@tiburd

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 6]
```diff
@@ -546,12 +547,26 @@ New attributes have now been introduced in the output of Get-CsOnlineUser when n
...
+
+If any information is required for a user that is not available in the output (when not using "-identity" parameter) then it can be obtained using the "-identity" parameter. Information for all users would be available using the "-identity" parameter until they are hard deleted.
+
+If Guest users and SoftDeletedUsers are not required in the output then they can be filtered out by using filter on AccountType and SoftDeletionTimestamp respectively.
+
+### -------------------------- Example --------------------------
+```
+Get-csOnlineUser -Filter {AccountType -ne 'Guest'}
+Get-csOnlineUser -Filter {SoftDeletionTimestamp -eq $null}
+```
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:565]:
```suggestion
If Guest users and SoftDeletedUsers are not required in the output then they can be filtered out by using filter on AccountType and SoftDeletionTimestamp respectively. For example:
Get-csOnlineUser -Filter {AccountType -ne 'Guest'}
Get-csOnlineUser -Filter {SoftDeletionTimestamp -eq $null}
```

Comment (dariomws) [reply skype/skype-ps/skype/Get-CsOnlineUser.md:565]: I don't think that example format is expected in this section.

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by dariomws** [comment]
@isrumnon, try removing the example format in the Notes section.

**Comment by isrumnon** [comment]
> @isrumnon, try removing the example format in the Notes section.

I think that worked, thanks!

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 7]
```diff
@@ -546,12 +547,22 @@ New attributes have now been introduced in the output of Get-CsOnlineUser when n
 - UserValidationErrors                   
 - WhenCreated    
      
-Unlicensed Users: Unlicensed users would show up in the output for 30 days post-license removal.
-Soft deleted users: These users will be displayed in the output of Get-CsOnlineUser and the TAC Manage Users page by default with SoftDeletionTimestamp set to a value.
+The updates below are now applicable to the output in scenarios where "-identity" parameter is not used:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:550]:
```suggestion
The following updates are now applicable to the output in scenarios where "-identity" parameter is not used:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @isrumnon, one more minor change, please (we do not use the terms "above" and "below" in Microsoft docs. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips gentle reminder.

**Review by get-itips — APPROVED** [review]

**Comment by isrumnon** [comment]
@tiburd @yogkumgit can we merge this now?

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by isrumnon** [comment]
Hi @yogkumgit @JohanFreelancer9 need some help. The published changes in this are not consistent with what the preview was showing me. Any idea what I can do to fix this?

image
Production

image
Preview

**Comment by yogkumgit** [comment]
@tiburd Please help to @isrumnon Thanks

**Comment by tiburd** [comment]
@isrumnon 
Hi, Isaiah,
I'll send you email.
Thanks!

**Pull request merged**